lib/cli/commands/listcurrencies.ts

```diff
@@ -0,0 +1,47 @@
+import { callback, loadXudClient } from '../command';
+import { Arguments } from 'yargs';
+import colors from 'colors/safe';
+import Table, { HorizontalTable } from 'cli-table3';
+import { ListCurrenciesRequest, ListCurrenciesResponse } from '../../proto/xudrpc_pb';
+
+const HEADERS = [
+  colors.blue('Ticker'),
+  colors.blue('Digits'),
+  colors.blue('Identifier'),
```

Comment:
Since this will only ever contain raiden token addresses (which is fine I think), it'd probably be better to just call this `Token address`.